packages/ui/core-components/src/lib/organisms/layout/header/SearchButton.css

```diff
@@ -1,9 +1,9 @@
 .DocSearch-Button {
-	@apply bg-white hover:bg-gray-50 transition-colors duration-200 hover:text-gray-800 rounded-md flex gap-16 cursor-pointer py-1 pl-2 pr-1 shadow-sm sm:text-xs text-gray-600 border font-sans font-medium  items-center;
+	@apply bg-white hover:bg-gray-50 transition-colors duration-200 hover:text-gray-800 rounded-md flex gap-16 cursor-pointer py-1 pl-2 sm:pr-1  pr-20 shadow-sm sm:text-xs text-gray-600 border font-sans font-medium  items-center;
```

Comment:
when on `xs` size the text displaying `Search` is larger than on `md+` - Seems odd